spec/duplicates_spec.rb

```diff
@@ -1,3 +1,4 @@
+require 'spec_helper'
```

Comment:
One last thing, what do you think about adding `--require spec_helper` to the `.rspec` instead of manually requiring it in each test?